.chloggen/feat_ecs-format.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
```

Comment:
Others LGTM, But I have a question, is this a breaking for users who have already configured  with `ecs mode`?

Reply:
@JaredTan95 thats actually a very good point. If a user explicitly has `ecs` specified in their config this would breaking change as the format of their logs would change. I will change the release notes to reflect this.

Reply:
Fixed.